homework_js2.js

```diff
@@ -0,0 +1,18 @@
+//зробити змінну з значенням "We are good"
+let tempValue = 'We are good'
+console.log(tempValue)
+
+
+//створити 2 зміні зі значеннями ''Hello" і "World", створити команду для виведення в консоль цих слів разом
+let firstWord = 'Hello'
+let secondWord = "World"
+console.log(firstWord, secondWord)
```

Comment:
code should be like 'Hello World"

Reply:
made changes in the code, validate it, please